dapps/marketplace/src/components/Error404.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+
+const Error404 = props => {
+  return (
+    <div className="error-404">
+      <div className="container">
+        <div className="row">
```

Comment:
I think we can remove the `row` and `col-12` and put the `props.children` directly in the container